demonstrations/tutorial_noisy_circuit_optimization.py

```diff
@@ -0,0 +1,519 @@
+r"""
+Optimizing noisy circuits with Cirq
+===================================
+
+.. meta::
+    :property="og:description": Learn how noise can affect the optimization and training of quantum computations.
+    :property="og:image": https://pennylane.ai/qml/_images/noisy_circuit_optimization_thumbnail.png
+
+.. figure:: ../demonstrations/noisy_circuit_optimization/noisy_qubit.png
+    :align: center
+    :width: 90%
+
+Until we have fault-tolerant quantum computers, 
+we will have to learn to live with noise. There are lots of exciting 
+ideas and algorithms in quantum computing and quantum machine learning, 
+but how well do they survive the reality of today's noisy devices? 
+
+Background
+----------
+
+Quantum pure-state simulators are great and readily available in 
+a number of quantum software packages.
+They allow us to experiment, prototype, test, and validate algorithms
+and research ideas---up to a certain number of qubits, at least.
+
+But present-day hardware is not ideal. We're forced to confront 
+decoherence, bit flips, amplitude damping, and so on. Does the 
```

Comment:
? Also did a web search, but didn't chance upon any good resources for bit flips and amplitude damping.

```suggestion
But present-day hardware is not ideal. We're forced to confront 
`decoherence <https://en.wikipedia.org/wiki/Quantum_decoherence>`__, bit flips, amplitude damping, and so on. Does the 
```